packages/scm/src/browser/scm-contribution.ts

```diff
@@ -98,6 +98,9 @@ export class ScmContribution extends AbstractViewContribution<ScmWidget> impleme
         });
         this.scmService.onDidChangeSelectedRepositories(repository => {
             if (repository) {
+                if (this.scmService.repositories.length === 1) {
+                    return;
```

Comment:
Should not we explicitly remove `CHANGE_REPOSITORY`?

Reply:
fixed